### PR TITLE
chore: fix swb-ui package

### PR DESCRIPTION
**Pull request by insignias**
Issue #, if available:

Description of changes:
1. Fix rush config in swb-ui/infrastructure package
2. Fix swb-ui directory structure

Checklist:

* [ ] Have you successfully deployed to an AWS account with your changes?
* [ ] Have you written new tests for your core changes, as applicable?

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: workbench-core/example/infrastructure/README.md [hunk 1]
```diff
@@ -5,7 +5,7 @@
 # Code Coverage
 | Statements                  | Branches                | Functions                 | Lines             |
 | --------------------------- | ----------------------- | ------------------------- | ----------------- |
-| ![Statements](https://img.shields.io/badge/statements-100%25-brightgreen.svg?style=flat) | ![Branches](https://img.shields.io/badge/branches-100%25-brightgreen.svg?style=flat) | ![Functions](https://img.shields.io/badge/functions-100%25-brightgreen.svg?style=flat) | ![Lines](https://img.shields.io/badge/lines-100%25-brightgreen.svg?style=flat) |
+| ![Statements](https://img.shields.io/badge/statements-Unknown%25-brightgreen.svg?style=flat) | ![Branches](https://img.shields.io/badge/branches-Unknown%25-brightgreen.svg?style=flat) | ![Functions](https://img.shields.io/badge/functions-Unknown%25-brightgreen.svg?style=flat) | ![Lines](https://img.shields.io/badge/lines-Unknown%25-brightgreen.svg?style=flat) |
```
Comment (kpark277) [inline workbench-core/example/infrastructure/README.md:8]: Is it okay that these are now all Unknown?

File: solutions/swb-ui/infrastructure/.eslintignore [hunk 2]
```diff
@@ -0,0 +1,7 @@
+.eslintrc.js
+node_modules
+lib
+temp
+cdk.out
+build
+#redirectFunction.js
```
Comment (kpark277) [inline solutions/swb-ui/infrastructure/.eslintignore:7]: Should this be deleted?

**Review by bhudnell — COMMENTED** [review]
Should the `out` directory be `.gitignore`'d?

File: workbench-core/example/infrastructure/README.md (hunk 1 above)
Comment (insignias) [reply workbench-core/example/infrastructure/README.md:8]: @kpark277 Missed to update this...I will fix it...but it shouldn't matter, we have a workflow to automatically update the README files after the PR is merged to develop

**Comment by insignias** [comment]
> Should the `out` directory be `.gitignore`'d?

Good catch, forgot to update the path in .gitignore

**Review by kpark277 — APPROVED** [review]

**Review by bhudnell — DISMISSED** [review]

File: solutions/swb-ui/infrastructure/package.json [hunk 3]
```diff
@@ -17,38 +17,52 @@
...
+    "eslint-plugin-security": "^1.4.0",
+    "istanbul-badges-readme": "1.8.1",
     "license-check-and-add": "^4.0.5",
     "license-checker": "^25.0.1",
+    "npm-package-json-lint": "^6.3.0",
+    "npm-package-json-lint-config-default": "^5.0.0",
     "sort-package-json": "^1.57.0",
     "source-map-support": "^0.5.16",
-    "ts-jest": "^27.1.3",
-    "ts-node": "^10.4.0",
```
Comment (pbabz) [inline solutions/swb-ui/infrastructure/package.json:51]: Removing these (and the jest above) breaks me locally

**Review by pbabz — CHANGES_REQUESTED** [review]

File: solutions/swb-ui/infrastructure/package.json (hunk 3 above)
Comment (insignias) [reply solutions/swb-ui/infrastructure/package.json:51]: @pbabz You shouldn't need these. pushing an update. Can you please try it out again

File: solutions/swb-ui/infrastructure/package.json [hunk 4]
```diff
@@ -17,38 +17,52 @@
...
     "@rushstack/heft-jest-plugin": "^0.3.28",
     "@rushstack/heft-node-rig": "^1.10.5",
+    "@types/heft-jest": "1.0.2",
     "@types/jest": "^27.4.1",
     "@types/js-yaml": "^4.0.5",
     "@types/node": "^14",
     "@types/prettier": "2.6.3",
     "aws-cdk": "^2.12.0",
     "aws-cdk-lib": "^2.29.1",
+    "cdk-nag": "^2.12.25",
```
[the remark this answers is not in the document]
Comment (insignias) [reply solutions/swb-ui/infrastructure/package.json:53]: @pbabz eventually we will :) but cleaning it up for now !

**Review by kpark277 — APPROVED** [review]

**Review by bhudnell — APPROVED** [review]

File: solutions/swb-ui/infrastructure/.eslintignore [hunk 5]
```diff
@@ -0,0 +1,7 @@
+.eslintrc.js
+node_modules
+lib
+temp
+cdk.out
+build
+src/redirectFunction.js
```
Comment (htztomic) [inline solutions/swb-ui/infrastructure/.eslintignore:7]: reason why we're ignoring this file?

Comment (insignias) [reply solutions/swb-ui/infrastructure/.eslintignore:7]: This conversation is resolved, answered on slack !

**Review by htztomic — APPROVED** [review]
LGTM, src/redirectFunction.js needs to be converted to ts

**Pull request merged**